src/coprocessor/dag/expr/fncall.rs

```diff
@@ -168,6 +168,8 @@ impl FnCall {
             ScalarFuncSig::IfDecimal |
             ScalarFuncSig::IfTime |
             ScalarFuncSig::IfDuration => (3, 3),
+
+            _ => unimplemented!(),
```

Comment:
I think panic is not a good idea.

Reply:
I think `unimplemented!()` will panic here. doesn't it?